lib/PuppeteerSharp/Launcher.cs

```diff
@@ -156,7 +156,7 @@ public async Task<Browser> ConnectAsync(ConnectOptions options)
 
                 _connection = await Connection.Create(options.BrowserWSEndpoint, connectionDelay, keepAliveInterval, _loggerFactory).ConfigureAwait(false);
 
-                return await Browser.CreateAsync(_connection, options, null, () =>
+                return await Browser.CreateAsync(_connection, options.IgnoreHTTPSErrors, options.AppMode, null, () =>
```

Comment:
Sorry for not picking everything at once. But here we should send `true` instead of appMode 
https://github.com/kblok/puppeteer/blob/51645932b7dc3c3f3d958deb667c22b3519bcc91/lib/Launcher.js#L233